build/docker/marti-dev/web-files/root/etc/nginx/conf.d/default.conf

```diff
@@ -5,7 +5,7 @@ server {
 
     root /usr/share/nginx/html/dice;
 
-    index index.html index.php;
+    index register.php;
 
     location / {
```

Comment:
Since you're updating this configuration anyway, can you please also remove the root location rule?  It was already removed from the staging configuration when we determined it wasn't necessary when enabling HTTPS.